agent/tool-scripts/postprocess/samples/jmap/jmap-stdout.txt

```diff
@@ -0,0 +1,3 @@
+1526474998,java-1,131.696
+1526474998,java-2,131.000
+1526474998,532,131.001
```

Comment:
What's the format here?

Reply:
Here I generate it differently in order to avoid output processing, so that we can write the results faster, by reasonable performance.

so I've just throw the output to a new line, each new line represents process id or name.